### PR TITLE
New service: DiscordServices

### LINKED ISSUE #55: New Service: DiscordServices
Website: https://discordservices.net/
Docs: https://discordservices.net/docs/api

### PR DESCRIPTION
**Pull request by EndBug**
Closes #55

### CODE REVIEW

File: src/Interface/Lists/DiscordServices.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+   * @property {string} command The command (include the prefix)
+   * @property {string} desc The description for your command
+   * @property {string} category The category of your command
+   */
+
+  /**
+   * Posts commands info to your bot page
+   * @param {string} id The bot's ID
+   * @param {Array<DiscordServicesCommandInfo>} commands The command info to post
+   * @param {string} content The content of the post
```
Comment (Snazzah) [inline src/Interface/Lists/DiscordServices.js:86]: Unused doc parameter

File: src/Interface/Lists/DiscordServices.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+   */
+  postCommands(id, commands) {
+    return this._request({
+      method: 'post',
+      url: `/bot/${Util.resolveID(id)}/news`,
```
Comment (Snazzah) [inline src/Interface/Lists/DiscordServices.js:91]: Wrong Endpoint

File: src/Interface/Lists/DiscordServices.js [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  }
+
+  /**
+   * Gets the widget URL for this bot.
+   * @param {IDResolvable} id The bot's ID
+   * @param {Query} [query] The query string that will be used in the request
+   * @param {string} [smallWidget] The sub-path name to turn the widget into a badge (i.e. owner)
+   * @returns {string}
+   */
+  getWidgetURL(id, query, smallWidget = null) {
```
Comment (Snazzah) [inline src/Interface/Lists/DiscordServices.js:104]: Does this exist for this service?

File: src/Interface/Lists/DiscordServices.js [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+  /**
+   * @typedef {Object} DiscordServicesCommandInfo
+   * @property {string} command The command (include the prefix)
+   * @property {string} desc The description for your command
+   * @property {string} category The category of your command
+   */
+
+  /**
+   * Posts commands info to your bot page
+   * @param {string} id The bot's ID
```
Comment (Snazzah) [inline src/Interface/Lists/DiscordServices.js:84]: Should be IDResolvable

File: src/Interface/Lists/DiscordServices.js [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+        title,
+        content,
+        error: false
+      }
+    }, { requiresToken: true });
+  }
+
+  /**
+   * @typedef {Object} DiscordServicesCommandInfo
+   * @property {string} command The command (include the prefix)
```
Comment (Snazzah) [inline src/Interface/Lists/DiscordServices.js:77]: Doc can probably be changed to "The command name including the prefix"

File: src/Interface/Lists/DiscordServices.js [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+          servers: Util.resolveCount(serverCount),
+          shards: shard.count
+        } :
+        { servers: Util.resolveCount(serverCount) }
+    });
+  }
+
+  /**
+   * Posts news to your bot page
+   * @param {string} id The bot's ID
```
Comment (Snazzah) [inline src/Interface/Lists/DiscordServices.js:58]: Should be IDResolvable

**Review by Snazzah — CHANGES_REQUESTED** [review]

**Comment by EndBug** [comment]
What a mess! Let's see if it's better now 😅

**Review by Snazzah — APPROVED** [review]

**Pull request merged**